.github/workflows/ci_cd.yml

```diff
@@ -53,41 +53,52 @@ jobs:
+      experimental_container_revn: ${{ steps.save-versions.outputs.experimental_container_revn }}
+      experimental_revn:  ${{ steps.save-versions.outputs.experimental_revn }}
     steps:
       - id: save-versions
         run: |
           parse_revns() {
-            revn=$1  # For example, 231
-            container_name=$2
-            license_name=$3
+            # This function accepts 2-3 arguments:
+            # $1 is the revision number (for example, 232)
+            # $2 is the name of the container revision number variable to be assigned (for example, stable_container_revn)
+            # $3 is the name of the revision number variable to be assigned (for example, experimental_revn)
+            #    This is required only for the experimental revn to set up the ANSYSCL<revn>_DIR environment variable
+            #    later in the workflow.
+          
+            # Get revision number from env variable or input (for manually run workflows)
+            # For example, revn=231
+            revn=$1
 
             # Create container_version from revn
-            container_version="${revn:0:2}.${revn:2}.0"  # For example, 23.1.0
-
-            # Set output variables
-            echo "$container_name=$container_version" >> $GITHUB_OUTPUT
-            echo "$license_name=$revn" >> $GITHUB_OUTPUT
+            # For example, container_version=23.1.0
+            container_version="${revn:0:2}.${revn:2}.0" 
+
+            # Set stable_container_revn or experimental_container_revn to the container_version
+            # These variables are saved in the "output" section above.
+            # For example, echo "stable_container_revn=23.1.0" >> $GITHUB_OUTPUT
+            echo "$2=$container_version" >> $GITHUB_OUTPUT
+
+            # Set experimental_revn to the revn
+            # For example, echo "experimental_revn=231" >> $GITHUB_OUTPUT
+            if [[ $3 == *"experimental"* ]]; then
+                echo "$3=$revn" >> $GITHUB_OUTPUT
+            fi
           }
 
-          # Use default revn values if the input values were not chosen.
-          # The input values are only selected in workflow_dispatch events.
-          if [[ -z "${{ inputs.default-revn }}" || -z "${{ inputs.test-revn }}" ]]; then
-            # If the event is a scheduled run, set the value to the DEV_REVN env var.
-            if ${{ github.event_name == 'schedule' }}; then
-              parse_revns "${{ env.DEV_REVN }}" "default" "default_lic"
-              parse_revns "${{ env.DEV_REVN }}" "test" "test_lic"
-            else
-              parse_revns "${{ env.DEFAULT_REVN }}" "default" "default_lic"
-              parse_revns "${{ env.TEST_REVN }}" "test" "test_lic"
-            fi
+          if ${{ github.event_name == 'schedule' }}; then
+              # Scheduled runs use the DEV_REVN environment variable
+              parse_revns "${{ env.DEV_REVN }}" "stable_container_revn"
```

Comment:
@klmcadams - instead of calling parse_revns twice can we pass stable & experimental revisions here and calculate stable_container_revn, experimental_container_revn and experimental_revn.